docs/about/release-notes.rst

```diff
@@ -17,6 +17,8 @@ Features
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+* Changes names of arithmetic functions to match numpy's names: ``add``, ``subtract``, ``multiply`` `#1999 <https://github.com/scipp/scipp/pull/1999>`_.
```

Comment:
Minor point, maybe list the exact names deprecated, for easy look up by users, can be done at some later point in any case.